task/s2i/0.4/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+add_sidecar_secure_registry
+
+# Add git-clone
+add_task git-clone latest
```

Comment:
there is some problem with git-clone latest on our CI infra, can you please use 0.7 version for the time being?

Reply:
I rebased the branch, is that enough?